apps/src/craft/code-connection/CraftVisualizationColumn.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import {default as GameButtons} from '../../templates/GameButtons';
+import BelowVisualization from '../../templates/BelowVisualization';
+import ProtectedVisualizationDiv from '../../templates/ProtectedVisualizationDiv';
+
+export default React.createClass({
```

Comment:
The answer is yes. Both options (class and createClass) are currently considered acceptable.